PlgxTools/Microsoft.Build.Plgx.xsd

```diff
@@ -3,8 +3,11 @@
  xmlns:msb="http://schemas.microsoft.com/developer/msbuild/2003"
  targetNamespace="http://schemas.microsoft.com/developer/msbuild/2003"
  elementFormDefault="qualified">
-  <xs:include schemaLocation="C:\Program Files (x86)\Microsoft Visual Studio 14.0\Xml\Schemas\1033\Microsoft.Build.xsd" />
-
+  <!-- <xs:include schemaLocation="C:\Program Files (x86)\Microsoft Visual Studio 14.0\Xml\Schemas\1033\Microsoft.Build.xsd" /> -->
+  
+  <!-- Support for Visual Studio 2017 Community -->
+  <xs:include schemaLocation="C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Xml\Schemas\1033\Microsoft.Build.xsd" />

```

Comment:
That is a good point, I did a bit of research beforehand, but I couldn't find anything like that. It is possible to have multiple include elements in it though. For example, line 6 could stay there uncommented but it then it will result in warning messages at compile time as it is not going to be able to find Microsoft Visual Studio 14.0 path if Visual Studio 2015 isn't installed. However, it will compile correctly as long as one of the includes path exists. Another idea: maybe it is possible to create a conditional XSD schema structure to pick the right path based on the Visual Studio version. The problem with it is that the VS version needs to be read from the XSD file somehow.

Reply:
We also need this for cross-platform compatibility!